modules/apps/portal/portal-mail/src/main/java/com/liferay/portal/mail/sender/internal/MailSenderImpl.java

```diff
@@ -12,20 +12,20 @@
  * details.
  */
 
-package com.liferay.mail.service.impl;
+package com.liferay.portal.mail.sender.internal;
 
 import com.liferay.mail.kernel.model.Account;
 import com.liferay.mail.kernel.model.Filter;
 import com.liferay.mail.kernel.model.MailMessage;
-import com.liferay.mail.kernel.service.MailService;
 import com.liferay.mail.kernel.util.Hook;
 import com.liferay.portal.kernel.cluster.Clusterable;
 import com.liferay.portal.kernel.log.Log;
 import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.mail.sender.MailSender;
 import com.liferay.portal.kernel.messaging.DestinationNames;
```

Comment:
Kept the DestinationNames instead of making a new constant class
https://github.com/liferay-core-infra/liferay-portal/pull/768#discussion_r890843216